string-manipulation-1/src/is-vowel.ts

```diff
@@ -0,0 +1,10 @@
+/* exported isVowel */
+function isVowel(char: string): boolean {
+  const vowels = ['a', 'e', 'i', 'o', 'u'];
+  for (let i = 0; i < vowels.length; i++) {
+    if (char.toLowerCase() === vowels[i]) {
+      return true;
+    }
+  }
+  return false;
```

Comment:
Looks good but this could also be a case for a `switch` statement since we are checking for specific cases.
```
switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
```